### PR TITLE
[14.0][IMP] project_task_add_very_high: add message in chatter

### PR DESCRIPTION
**Pull request by dessanhemrayev**
Added a message "priority:[old_priority] -> [new_priority]" to the chatter whenever the priority of a task is changed.

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This minimal change shouldn't qualify for co-authorship, and also introduce it in contributors may be too much.

**Review by francesco-ooops — APPROVED** [review]

File: project_task_add_very_high/models/project_task.py [hunk 1]
```diff
@@ -7,4 +7,6 @@
 class ProjectTask(models.Model):
     _inherit = "project.task"
 
-    priority = fields.Selection(selection_add=[("2", "High"), ("3", "Very High")])
+    priority = fields.Selection(
+        selection_add=[("2", "High"), ("3", "Very High")], tracking=True
```
Comment (astirpe) [inline project_task_add_very_high/models/project_task.py:11]: This new functionality should be mentioned in the readme

Comment (dessanhemrayev) [reply project_task_add_very_high/models/project_task.py:11]: Thanks, I'll add it now)

**Review by astirpe — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@pedrobaeza good now?

**Comment by pedrobaeza** [comment]
Still adding 2 lines of contributors for a simple `tracking=True` seems too much for me.

**Comment by francesco-ooops** [comment]
@dessanhemrayev please remove contribution :)

**Comment by francesco-ooops** [comment]
@pedrobaeza go? :)

**Review by pedrobaeza — APPROVED** [review]
Thanks for the changes.

/ocabot merge minor

**Pull request merged**